Templates/Databases/MicrosoftSQL/Query Microsoft SQL Database and return JSON GET/long-description.md

```diff
@@ -0,0 +1,54 @@
+This template process receives an **HTTP request**, queries data from a **MicrosoftSQL database** based on parameters given in the HTTP request, and returns the queried data to the caller as **JSON**. 
+
+![Template](assets/Query_Microsoft_SQL_Database_and_return_JSON_GET.svg)
+Templates\Databases\MicrosoftSQL\Query Microsoft SQL Database and return JSON GET\assets\Query_Microsoft_SQL_Database_and_return_JSON_GET.svg
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The **MicrosoftSQL database** user has permissions to read data from the specified table.
+
+
+# Implementation and Usage Notes
+
+This template process returns **JSON** data based on the execution of an SQL SELECT query formed using parameters given in the **HTTP request**.
+The HTTP request path parameters specify a column and value to be filtered, for example "email" and "my.email@domain.com". The first parameter defines the column, and the second the value. The process will then return all matching rows as **JSON**.
+
+The column name given in the HTTP request parameter is validated by only allowing alphanumeric characters and underscores. This is because column names cannot be passed as query parameters to the SQL query to be executed, and thus aren't automatically validated.
+If the validation is not passed, the template will throw an exception.
+
+The Process Variable ConnectionString determines the **Microsoft SQL** database to be connected to, and the Process Variable TableName specifies the table to be queried.
+
+By default, the HTTP trigger is set to use [API key](https://docs.frends.com/en/articles/2206706-api-keys) authentication.
+
+If this template process is to be used as a part of a larger API, the trigger can be changed into an [API trigger](https://docs.frends.com/en/articles/2206741-api-trigger).
+
+**Example HTTP request URL**
+
+`https://myfrendsagent.frendsapp.com/queryDatabase/email/my.email@domain.com`
+
+**Example return value**
+```
+{
+    "data": [
+        {
+            "id": 1,
+            "name": "my-name",
+            "email": "my.email@domain.com"
```

Comment:
change domain.com into example.com. Domain.com is actually an existing business and example.com is specifically for examples in docs. (please also find other examples and change to example.com if needed)